src/plugins/index_pattern_management/public/components/edit_index_pattern/indexed_fields_table/components/table/table.tsx

```diff
@@ -185,6 +197,15 @@ export class Table extends PureComponent<IndexedFieldProps> {
             />
           </span>
         ) : null}
+        {field.customLabel !== field.name && (
+          <div>
+            <EuiToolTip content={customLabelDescription}>
+              <EuiBadge iconType="flag" iconSide="left">
+                {field.customLabel}
+              </EuiBadge>
+            </EuiToolTip>
```

Comment:
Added flag and tooltip:

<img width="321" alt="Bildschirmfoto 2020-08-31 um 13 48 19" src="https://user-images.githubusercontent.com/463851/91718449-a63da880-eb93-11ea-8198-6b0c1ae3447a.png">

@andreadelrio Note that the `EuiBadge` also shows a sort of popover 
FYI @gchaps, is this wording fine, or would you suggest something different ?

Reply:
@kertal Looks great!